zoab_kapoor.cfdg

```diff
@@ -0,0 +1,8 @@
+startshape serpinski
+
+rule serpinski {
+CIRCLE { }
+serpinski { y .255 s .5 b .1 sat .2 }
+serpinski { x -.25 y -.144 s .5 b .1 sat .5 }
+serpinski { x .25 y -.144 s .5 b .1 sat .8 }
+}
```

Comment:
How did you choose the saturation and brightness levels? Just trying random guesses and seeing what looked good?